### PR TITLE
Issue #4 as well as code cleanup

### PR DESCRIPTION
**Pull request by Teiem**

### CODE REVIEW

File: scripts/startpage.js [hunk 1]
```diff
@@ -55,28 +62,9 @@ document.getElementById('search-form').addEventListener('submit', e => {
 		if (Object.keys(dict_services).includes(key)) {
 			let url = dict_services[key].split('|')[1];
 			window.location = `${url}`;
-		} else if (Object.keys(dict_sites).includes(key)) {
-			window.location = `https://${dict_sites[key]}`;
```
Comment (kittenparry) [inline scripts/startpage.js]: Removal of this `else if` block breaks functionality and makes it unable to visit sites mentioned on left. Eg. `i` for `instagram`, `y` for `youtube` etc.

**Comment by kittenparry** [comment]
Also I appreciate that `curSearch` variable. It takes a great care of the repetitive mess I had for the second event listener.

**Comment by Teiem** [comment]
> So, I could either accept it and make the change myself or wait for you to put that small block again.

Ups, didnt noticed I deleted it - If I find out how to change it I will - you should also have the right to change the request.

**Comment by kittenparry** [comment]
Thanks again for taking your time and doing this.. Merging.

**Pull request merged**